config/application.rb

```diff
@@ -9,7 +9,7 @@
 module Earthworks
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults 6.1
     config.autoloader = :classic # Our helper overrides don't work under the new loader
```

Comment:
Unfortunately we still can't use zeitwerk.. 🤷

Reply:
Upstream issue filed https://github.com/geoblacklight/geoblacklight/issues/996 . Seems like we need to modify how things behave are loaded up there to make the magic happen here.